src/SFA.DAS.PSRService.Domain/Entities/Period.cs

```diff
@@ -1,77 +1,80 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.Linq;
 
 namespace SFA.DAS.PSRService.Domain.Entities
 {
-    public class Period
+    public class Period : IEquatable<Period>
     {
-        private DateTime _periodDateTime { get; set; }
-        private int _startYear => GetReportPeriodStartYear();
-        private int _endYear => GetReportPeriodEndYear();
+        public TwoThousandsCommonEraYear StartYear { get; }
+
+        public TwoThousandsCommonEraYear EndYear { get; }
+
+        public string FullString => $"1 April {StartYear.AsFourDigitString} to 31 March {EndYear.AsFourDigitString}";
 
-        public string StartYear => (_startYear).ToString();
-        public string EndYear => (_endYear).ToString();
-        public string FullString => GetCurrentReportPeriodName();
-        public bool IsCurrent => IsCurrentPeriod();
         public string PeriodString => GetReportPeriod();
-        
-        public Period(DateTime period)
+
+        public static Period FromInstantInPeriod(DateTime instantInPeriod)
         {
-            _periodDateTime = period;
+            var endYear =
+                instantInPeriod.Month < 4
+                    ? instantInPeriod.Year - 1
+                    : instantInPeriod.Year;
+
+            return new Period(
+                startYear: TwoThousandsCommonEraYear.FromYearAsNumber(endYear - 1),
+                endYear: TwoThousandsCommonEraYear.FromYearAsNumber(endYear));
         }
 
-        public Period(string period)
+        public static Period ParsePeriodString(string periodString)
         {
-            var year = ConvertPeriodStringToYear(period);
+            if (string.IsNullOrWhiteSpace(periodString))
```

Comment:
Validation - there are two checks resulting in the same message; the messages in the ArgumentExceptions could be more specific and informative.